etc/ProdOfflineConfiguration.py

```diff
@@ -1176,6 +1166,22 @@
 ### Parking and Scouting PDs                         ###
 ########################################################
 
+DATASETS = ["ScoutingPFRun3"]
+for dataset in DATASETS:
+    addDataset(tier0Config, dataset,
+               do_reco=True,
+               scenario=ppScenario)
```

Comment:
in the gDoc I see:
> ScoutingPFRun3 -> scouting = no reco (ScoutingPFMonitor should go in the usual reco)
(about parking see PPD [presentation](https://indico.cern.ch/event/1179710/#41-b-parking-trigger-developme)) (it should be repacked, no reconstruction  and double copy - which is the default behaviour)

So maybe do_reco should be false?

Reply:
Yes my mistake. Its fixed now